software/donetick.yml

```
@@ -0,0 +1,17 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Nodejs
+  - Docker
+tags:
+  - Task Management & To-do Lists
+  - Automation
+  - Personal Dashboards
+
+depends_3rdparty: true
```

Where does it depend on a 3rd party? I couldn't find it when I quickly looked over the repo.
```suggestion
depends_3rdparty: true
```

---

I was not sure if 3rd_party here mean like not having zero dependency on other libraries :) i clearly miss read it

---

oh got it! Thank you @Rabenherz112